inference/worker/work.py

```diff
@@ -90,6 +103,11 @@ def handle_work_request(
 
     model_config = worker_config.model_config
 
+    safety_request = inference.SafetyRequest(prompt)
+    safety_response = get_safety_server_response(safety_request)
```

Comment:
should this be optional?

Reply:
I think we can already prepare by e.g. having a safety level 0 to 9, and for now we only send 0 or 9 and we're free to extend these later, but at least we won't have to change data types